client/src/pages/Login/Login.tsx

```diff
@@ -14,14 +17,25 @@ import {
   Title,
 } from "./Login.styles";
 
-const Login = () => {
-  const [email, setEmail] = useState<string>("");
-  const [password, setPassword] = useState<string>("");
+const Loginschema = yup.object().shape({
+  email: yup.string().email().required(),
+  password: yup.string().min(8).max(20).required().matches(/^[A-Za-z]+$/i)
+})
 
-  const handleSubmit = (e: React.FormEvent) => {
-    // code to validate login and password
-    e.preventDefault();
-  };
+interface LoginInputs {
+  email: string;
+  password: string;
+}
+
+export default function Login() {
+
+  const { register, handleSubmit, formState: {errors} } = useForm<LoginInputs>({
+    resolver: yupResolver(Loginschema),
+  });
+  // @ts-ignore 
```

Comment:
We don't want to have any ts-ignores here. Let's figure out what is erroring.

Reply:
we can remove the ts-ignore when we put something in the function. I kept getting an error since data was not being called anywhere